app/src/main/java/com/nervousfish/nervousfish/data_objects/Contact.java

```diff
@@ -11,15 +11,21 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.List;
+import java.util.Objects;
 
 /**
  * A Java object representation (POJO) of a contact.
  */
+@SuppressWarnings({"PMD.ImmutableField", "PMD.UselessParentheses"})
+//1. We don't want the field iban to be final.
+//2. The parentheses on line 125 are not useless.
 public final class Contact implements Serializable {
 
     private static final long serialVersionUID = -4715364587956219157L;
     private final String name;
     private final List<IKey> keys = new ArrayList<>();
+    @android.support.annotation.Nullable
+    private String iban;
```

Comment:
We discussed this over Whatsapp, and we want the IBAN to be optional. That is why the attribute can't be final. The annotation can be removed though.